robot_sdk/wrappers/ec_wrapper.py

```
@@ -199,3 +221,53 @@ def move_trajectory(self, waypoints: List[List[float]]) -> bool:
+                return False
+            if not isinstance(angle, (int, float)):
+                print("Error: Angle must be a number")
+                return False
+            joint_angles = self.client.getjointangles()
+            if joint_angles is None:
+                print("Error: Unable to get joint angles. Check robot connection.")
+                return False
+            angle_rad = np.deg2rad(angle)
+            joint_angles[joint_index] += angle_rad
+            self.client.movej(joint_angles=joint_angles)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)